credentials/google/google.go

```diff
@@ -76,6 +76,26 @@ func NewComputeEngineCredentials() credentials.Bundle {
 	return bundle
```

Comment:
Change `NewComputeEngineCredentials` to call the new function

Reply:
Done